### PR TITLE
improve dx

### PR DESCRIPTION
**Pull request by cleemansen**

### CODE REVIEW

**Comment by cleemansen** [comment]
- [ ] Docker just works

**Comment by cleemansen** [comment]
bring back possibility to inject (MoveApps) settings to the shiny-module (app-configuration)

**Comment by andreakoelzsch** [comment]
> bring back possibility to inject (MoveApps) settings to the shiny-module (app-configuration)

is that a question or reminder? why is this necessary?

**Comment by cleemansen** [comment]
> > bring back possibility to inject (MoveApps) settings to the shiny-module (app-configuration)
> 
> is that a question or reminder? 
it is a reminder

> why is this necessary?
to support the MoveApps feature _Local File Upload_

File: ShinyModule.R [hunk 1]
```diff
@@ -2,25 +2,32 @@ library("shiny")
...
-# logger.fatal() -> logger.trace()
+# one can use the function from the logger.R file:
+# logger.fatal(), logger.error(), logger.warn(), logger.info(), logger.debug(), logger.trace()
 
 shinyModuleUserInterface <- function(id, label) {
   ns <- NS(id) ## all IDs of UI functions need to be wrapped in ns()
+  # showcase to access a file ('auxiliary files') that is 
+  # a) provided by the app-developer and 
+  # b) can be overridden by the workflow user.
+  fileName <- paste0(getAppFilePath("yourLocalFileSettingId"), "sample.txt")
```
Comment (cleemansen) [inline ShinyModule.R:13]: Showcase LOCAL FILE

image

**Comment by cleemansen** [comment]
> > > bring back possibility to inject (MoveApps) settings to the shiny-module (app-configuration)
> > 
> > 
> > is that a question or reminder?
> 
> it is a reminder
> 
> > why is this necessary?
> 
> to support the MoveApps feature _Local File Upload_

This is NOT correct. No need for app settings in order to use LOCAL FILE feature. Will revert it.. sorry for the confusion.

**Review by annescharf — APPROVED** [review]

**Pull request merged**